Extension/src/common.ts

```diff
@@ -333,6 +333,15 @@ export function isArrayOfString(input: any): input is string[] {
     return isArray(input) && input.every(isString);
 }
 
+// Validates whether the given object is a valid mapping of key and value type.
+// EX: {"key": true, "key2": false} should return true for keyType = string and valueType = boolean.
+export function isValidMapping(value: any, isValidKey: (key: any) => boolean, isValidValue: (value: any) => boolean): value is object {
+    if (isObject(value)) {
```

Comment:
There appears to be an issue with the pre-existing isObject function -- it returns `true` for `null` which may be unexpected:
![image](https://github.com/user-attachments/assets/9ab96c5e-65bd-44d5-ac93-36d72c85aa20)

Reply:
I believe I fixed `isObject` as I noticed the same thing.